addons/website_event_booth/static/src/xml/event_booth_registration_templates.xml

```diff
@@ -0,0 +1,31 @@
+<templates>
+
+    <t t-name="event_booth_registration_complete">
+        <div class="col-12 p-5">
+            <div class="row mb-3">
+                <div class="col-12">
+                    <h3>Booth Registration completed!</h3>
+                    <span class="h4 text-muted" t-esc="event_name"/>
+                </div>
+            </div>
+            <div class="d-flex flex-column">
+                <span t-if="contact.name" class="font-weight-bold">
```

Comment:
You could t-esc directly in the span to save some lines.

`<span t-if="contact.name" t-esc="contact.name" class="font-weight-bold"/>`